bin/i18n/resources/dashboard/courses.rb

```diff
@@ -1,69 +1,29 @@
-require 'fileutils'
-require 'json'
-
 require_relative '../../i18n_script_utils'
-require_relative '../../redact_restore_utils'
+
+Dir[File.expand_path('../courses/**/*.rb', __FILE__)].sort.each {|file| require file}
 
 module I18n
   module Resources
     module Dashboard
       module Courses
-        I18N_SOURCE_FILE_PATH = CDO.dir(File.join(I18N_SOURCE_DIR, 'dashboard/courses.yml')).freeze
-
-        # Merging dashboard/config/courses/*.course with courses.yml
-        # These files contain resources used by UnitGroup (used in the landing pages of full courses).
-        # https://studio.code.org/courses/csd-2021
-        # All other resources used in Unit come from scrip_json files (used in the landing pages for each Unit).
-        # https://studio.code.org/s/csd1-2021
-        def self.sync_in
-          puts 'Preparing dashboard courses'
-
-          # Currently, csd is the only fully translatable course that has resources in this directory
-          translatable_courses = %w(csd)
-          resources = {}
-          Dir[CDO.dir('dashboard/config/courses/*.course')].each do |file|
-            course_json = JSON.load_file(file)
-            course_properties = course_json['properties']
-
-            next if course_json['resources'].empty?
-            next unless translatable_courses.include?(course_properties['family_name'])
+        # Currently, csd is the only fully translatable course that has resources in this directory
+        TRANSLATABLE_COURSES = %w[csd].freeze
 
-            course_json['resources'].each do |resource|
-              # resoruce.rb uses Services::GloballyUniqueIdentifiers.build_resource_key to create resource keys as follow
-              # resource.key / resource.course_version.course_offering.key / resource.course_version.key
-              # resource_key is used to localize the resources.
-              resource_key = [resource['key'], course_properties['family_name'], course_properties['version_year']].join('/')
-              resources.store(resource_key, {'name' => resource['name'], 'url' => resource['url']})
-            end
-          end
+        DIR_NAME = 'dashboard'.freeze
+        FILE_NAME = 'courses.yml'.freeze
+        ORIGIN_I18N_FILE_PATH = CDO.dir('dashboard/config/locales/courses.en.yml').freeze
+        I18N_SOURCE_FILE_PATH = CDO.dir(I18N_SOURCE_DIR, DIR_NAME, FILE_NAME).freeze
```

Comment:
We do have `I18N_SOURCE_DIR_PATH = CDO.dir(File.join(I18N_SOURCE_DIR, 'dashboard')).freeze` in the main `dashboard.rb` module. Should we be using the existing variables?